api/views/image.py

```diff
@@ -3,30 +3,54 @@
 from rest_framework.generics import ListAPIView
 from mimetypes import guess_type
 from ..renderers.image import ImageRenderer
+from ..models.author import Author
+from ..models.content import Post
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.authentication import BasicAuthentication, TokenAuthentication
+from rest_framework.renderers import JSONRenderer
 import os
 
-# GET /author/images/:imageid
-class Images(ListAPIView):
+def get_img_path(img_id):
+    mimetype = guess_type(img_id)[0]
+    cur_dir = os.path.dirname(__file__)
+    return ('{0}/../../{1}'.format(cur_dir, img_id), mimetype)
+
+# GET /author/:authorid/image
+class AuthorImage(ListAPIView):
     """
     Returns an image.
     """
-    renderer_classes = (ImageRenderer, )
+    authentication_classes = (TokenAuthentication, BasicAuthentication, )
```

Comment:
Ooops, I think this more so applies to the PostImage viewset. This one shouldn't have any because I think we want profile images to be public.